pytripgui/app_logic/app_callbacks.py

```diff
@@ -309,33 +309,52 @@ def export_patient_dicom_callback(self, patient_item):
         logger.debug("DICOM export finished.")
         return True
 
-    def export_dose_voxelplan_callback(self, result_item):
-        # TODO docstring
+    def export_dose_voxelplan_callback(self, result_item: SimulationResultItem):
+        """
+        Export dose cube to Voxelplan format with the selected name.
```

Comment:
From the comment (docstring) it is not clear how `the selected name` is related to `result_item`.
Morever `result_item` suggest that is it some data structure to be returned or modified by this function.
I'd rather suggest `simulation_result` or `data_item` as the variable name

Reply:
I like the name simulation_result, as it's more descriptive than data_item in my opinion. I will change it.